types/node/index.d.ts

```diff
@@ -4881,7 +4882,7 @@ declare module "tls" {
         servername?: string;
         path?: string;
         ALPNProtocols?: Array<string | Buffer>;
-        checkServerIdentity?: (servername: string, cert: string | Buffer | Array<string | Buffer>) => any;
+        checkServerIdentity?: CheckServerIdentityFunction;
```

Comment:
Isn't it possible to write `checkServerIdentity?: typeof checkServerIdentity` here and avoid the extra type `CheckServerIdentityFunction`?
This would ensure that the function and the property here are always in sync.

Reply:
The only use of typeof in this file was in the Global interface definition, so it seemed like it wasn't expected syntax to use, so I followed the lead of LookupFunction.

If you think using typeof directly and removing the type declaration is kosher in this file, then I'll fix :D

Reply:
The reason why I used a dedicated type for `LookupFunction`  instead `typeof dns.lookup` is that `dns.lookup` offers more options that the variant used by sockets and I don't think there is a `typeof` operand to select just the type of one specific overload.
But here we don't have this special case therefore I would suggest to go for `typeof` - except someone else (or CI) explains why not.

Reply:
Gotcha. Fixed :)